app/bundles/PluginBundle/Command/FetchLeadsCommand.php

```diff
@@ -105,17 +117,28 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $params['start'] = $startDate;
             $params['end']   = $endDate;
             $params['limit'] = $limit;
+
             if (isset($supportedFeatures) && in_array('get_leads', $supportedFeatures)) {
                 if ($integrationObject !== null && method_exists($integrationObject, 'getLeads') && isset($config['objects'])) {
                     $output->writeln('<info>'.$translator->trans('mautic.plugin.command.fetch.leads', ['%integration%' => $integration]).'</info>');
-                    if (strtotime($startDate) > strtotime('-30 days')) {
-                        if (in_array('Lead', $config['objects'])) {
-                            $processed = intval($integrationObject->getLeads($params, null, $leads, [], 'Lead'));
+                    if (strtotime($startDate) > strtotime('-30 days') || $input->getOption('fetch-all')) {
+
+                        //Handle case when integration object are named "Contacts" and "Leads"
+                        $lead_object_name = 'Lead';
+                        if (in_array('Leads', $config['objects'])) {
+                            $lead_object_name = 'Leads';
                         }
-                        if (in_array('Contact', $config['objects'])) {
-                            $processed += intval($integrationObject->getLeads($params, null, $contacts, [], 'Contact'));
+                        $contact_object_name = 'Contact';
+                        if (in_array('Contacts', $config['objects'])) {
+                            $contact_object_name = 'Contacts';
```

Comment:
Use camelCase please, per our code style. Here and on the previous lines where snake_case was used.